hazelcast/pom.xml

```diff
@@ -287,8 +295,7 @@
             <groupId>com.fasterxml.jackson.core</groupId>
             <artifactId>jackson-core</artifactId>
             <version>2.9.7</version>
-            <scope>provided</scope>
-            <optional>true</optional>
+            <scope>compile</scope>
```

Comment:
Minor: compile is the default scope.

Reply:
I thought explicit would be better. It was the same way with old minimal-jackson dependency.